src/themes/uswitch/theme.json

```diff
@@ -1525,6 +1525,15 @@
           "caret": {
             "variant": "compounds.accordion.base.caret"
           }
+        },
+        "titleIcon": {
+          "marginRight": "13px",
+          "height": 36,
+          "width": 32,
+          "icon": {
+            "height": "100%",
+            "width": "100%"
+          }
```

Comment:
If this is the same for all themes, can't we add it to the base component?

Reply:
Good point, I've moved it back to the component